utils/bitfield/src/rleplus/mod.rs

```diff
@@ -173,140 +176,110 @@ impl RlePlus {
             index = range.end;
         }
 
-        let (bytes, padding_zeros) = writer.finish();
-        let mut bitvec = BitVec::from(bytes);
-
-        // `bitvec` now may also contains padding zeros if the number of written
-        // bits is not a multiple of 8
-        for _ in 0..padding_zeros {
-            bitvec.pop();
-        }
-
         // no need to verify, this is valid RLE+ by construction
-        Self(bitvec)
+        Self(writer.finish())
     }
 
-    // Returns a byte slice of the bit field's contents.
+    /// Returns a byte slice of the bit field's contents.
     pub fn as_bytes(&self) -> &[u8] {
-        self.0.as_slice()
+        &self.0
     }
 
-    // Converts a bit field into a byte vector.
+    /// Converts a bit field into a byte vector.
     pub fn into_bytes(self) -> Vec<u8> {
-        self.0.into()
+        self.0
     }
 }
 
+#[macro_export]
+macro_rules! rleplus {
```

Comment:
can you maybe add a comment explaining that this takes in non-encoded bits and rle encodes them? Think my assumption at first was that the macro was just taking in encoded bits

Reply:
Yeah totally, I just forgot to document them